pkg/graph/resolvers/plan_favorite.go

```diff
@@ -0,0 +1,74 @@
+package resolvers
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/mint-app/pkg/authentication"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/storage"
+)
+
+//IsPlanFavorited checks if a model plan is a favorite.
+func IsPlanFavorited(logger *zap.Logger, principal authentication.Principal, store *storage.Store, modelPlanID uuid.UUID) (bool, error) {
+
+	favorite, err := PlanFavoriteGet(logger, principal, store, modelPlanID)
+	if err != nil {
+		return false, err
+	}
+	isFavorite := false
+	if favorite != nil {
+		isFavorite = true
+
+	}
```

Comment:
isFavorite := (favorite != nil)

Reply:
thanks for the suggestion, that is better